File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    
    int i;
    for(i=0;i<completion.size();i++) {
        if(participant[i]!=completion[i])
            break;
    }
    
    answer = participant[i];
    
    
    return answer;
}

File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        
        int i;
        String answer = "";
        
        Arrays.sort(participant);
        Arrays.sort(completion);
        
        for(i=0;i<completion.length;i++) {
            if(!participant[i].equals(completion[i])) {
                return participant[i];
            }
        }
        
        return participant[i];
    }
}